### ISSUE TITLE
Add godoc badge

### ISSUE THREAD

**Issue by ethernetdan** [redspread/spread#41]

**Comment by ethernetdan** [comment]
You can use this badge: [![](https://godoc.org/rsprd.com/spread?status.svg)](http://godoc.org/rsprd.com/spread)

with:
```md
[![](https://godoc.org/rsprd.com/spread?status.svg)](http://godoc.org/rsprd.com/spread)
```

**Comment by ethernetdan** [comment]
Closed by #44
